photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -550,6 +554,8 @@ final class Driver(val params: Params, val sparkContext: SparkContext, val logge
   /**
    * Select best model according to validation evaluator
    *
+   * TODO: the return type is just a monolithic string, which is not helpful for machines?
```

Comment:
OK - IMHO the string could ultimately be removed, in favor of a function that parses the parameters and metadata and outputs whatever is suitable for the logs. But you are right about the return type. I'll rephrase the TODO, and reformulate it as a question, so we don't lose track of this possible refactoring.